frontend/src/metabase/public/containers/PublicOrEmbeddedDashboard/PublicOrEmbeddedDashboardView.tsx

```diff
@@ -235,3 +240,21 @@ function getCurrentTabDashcards({
     dashcard => dashcard.dashboard_tab_id === selectedTabId,
   );
 }
+
+/**
```

Comment:
logic is moved from `frontend/src/metabase/dashboard/hooks/use-dashboard-url-params.ts`